source/vocab/display.jsonld

```diff
@@ -1436,14 +1435,186 @@
     },
     "formatters": {
       "termComponentList-format": {
+        "TODO": "fix vue-client and remove me",
         "@id": "termComponentList-format",
         "@type": "fresnel:Format",
         "fresnel:propertyFormatDomain": "termComponentList",
         "fresnel:valueFormat": {
           "fresnel:contentAfter": "--",
-         "fresnel:contentLast": ""
+          "fresnel:contentLast": ""
         }
       }
     }
+  },
+  "formatters": {
+    "termComponentList-format": {
+      "@id": "termComponentList-format",
+      "@type": "fresnel:Format",
+      "fresnel:propertyFormatDomain": "termComponentList",
+      "fresnel:valueFormat": {
+        "fresnel:contentAfter": "--",
+        "fresnel:contentLast": ""
+      }
+    },
+    "commaBeforeProperty-format": {
+      "TODO": "fullerFormOfName skrivs alltid med parenteser i libris nu, ska det vara så?",
+      "@id": "commaBeforeProperty-format",
+      "@type": "fresnel:Format",
+      "fresnel:propertyFormatDomain": [
+        "lifeSpan",
+        "familyName",
+        "givenName",
+        "name",
+        "marc:numeration",
+        "lifeSpan",
+        "marc:titlesAndOtherWordsAssociatedWithAName",
+        "fullerFormOfName"
+      ],
+      "fresnel:propertyFormat": {
+        "fresnel:contentBefore": ", ",
+        "fresnel:contentFirst": ""
+      }
+    },
+    "agent-place-format": {
+      "@id": "agent-place-format",
+      "@type": "fresnel:Format",
+      "fresnel:classFormatDomain": [
+        "Agent"
+      ],
+      "fresnel:propertyFormatDomain": [
+        "place"
+      ],
+      "fresnel:propertyFormat": {
+        "fresnel:contentBefore": " (",
+        "fresnel:contentFirst": "(",
+        "fresnel:contentAfter": ")",
+        "fresnel:contentLast": ")"
+      }
+    },
+    "ProvisionActivity-comma-format": {
+      "@id": "ProvisionActivity-comma-format",
+      "@type": "fresnel:Format",
+      "fresnel:classFormatDomain": [
+        "ProvisionActivity"
+      ],
+      "fresnel:propertyFormatDomain": [
+        "place",
+        "year",
+        "startYear",
+        "date",
+        "marc:publicationStatus"
+      ],
+      "fresnel:propertyFormat": {
+        "fresnel:contentBefore": ", ",
+        "fresnel:contentFirst": ""
+      }
+    },
+    "ProvisionActivity-endYear-format": {
+      "TODO": "endYear without startYear? fresnel:contentNoValue on startYear? but we only want it when endYear is present",
```

Comment:
And the opposite of course.
We need a way to create virtual properties in showProperties or something.
(this is not what fresnel:mergeProperties does. It just merges the values of multiple properties to a single one)